tss-esapi/src/context/tpm_commands/attestation_commands.rs

```diff
@@ -322,8 +322,47 @@ impl Context {
         ))
     }
 
+    /// Get the current time and clock from the TPM
+    ///
+    /// # Errors
+    /// * if the qualifying data provided is too long, a `WrongParamSize` wrapper error will be returned
+    pub fn get_time(
+        &mut self,
+        signing_key_handle: KeyHandle,
+        qualifying_data: Data,
+        signing_scheme: SignatureScheme,
+    ) -> Result<(Attest, Signature)> {
+        let mut timeinfo_ptr = null_mut();
+        let mut signature_ptr = null_mut();
+        ReturnCode::ensure_success(
+            unsafe {
+                Esys_GetTime(
+                    self.mut_context(),
+                    ESYS_TR_RH_ENDORSEMENT,
```

Comment:
In general I think one should avoid using the TSS constants directly. So I would suggest writing this as ```ObjectHandle::Endorsement.into()``` or something similar.